src/ReverseProxy/Delegation/HttpSysDelegator.cs

```diff
@@ -106,7 +106,7 @@ public void DelegateRequest(HttpContext context, DestinationState destination)
             {
                 Log.QueueNoLongerExists(logger, destination.GetHttpSysDelegationQueue(), destination.Model?.Config?.Address, ex);
 
-                // The target queue is gone, detach from it so we can try to re-attach
+                // The target queue is gone, detach from it, so we can try to re-attach
```

Comment:
```suggestion
                // The target queue is gone, detach from it so we can try to re-attach
```
I'm not a native speaker, but I don't think this is right.

Reply:
Maybe
```csharp
// The target queue is gone. 
// Detach from it, so we can try to re-attach.
```

Reply:
Yes, I think it is clearer with 2 sentences but I am not a native speaker either.